.github/workflows/renovate-automatic-approval.yml

```diff
@@ -6,18 +6,19 @@ on:
 jobs:
   approve-renovate-prs:
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
+      - name: Obtain token
+        id: obtainToken
+        uses: tibdex/github-app-token@v2
+        with:
+          private_key: ${{ secrets.RENOVATE_APPROVE_PRIVATE_KEY }}
+          app_id: ${{ secrets.RENOVATE_APPROVE_APP_ID }}
       - name: Install GitHub CLI
         run: sudo apt-get install -y gh
-
       - name: Approve Renovate PRs
         env:
-          GITHUB_TOKEN: ${{ secrets.RENOVATE_APPROVE_TOKEN }}
+          GITHUB_TOKEN: ${{ steps.obtainToken.outputs.token }}
         run: |
-          for pr in $(gh pr list --author homarr-renovate[bot] --json number --jq .[].number); do
+          for pr in $(gh pr list --author homarr-approvals[bot] --json number --jq .[].number); do
```

Comment:
This should stay the same as homarr-renovate is still the creator of the pull request